pkgs/top-level/all-packages.nix

```
@@ -27532,6 +27532,10 @@ with pkgs;
 
   filezilla = callPackage ../applications/networking/ftp/filezilla { };
 
+  fire = callPackage ../applications/audio/fire {
+    inherit (darwin.apple_sdk.frameworks) Cocoa WebKit CoreServices DiscRecording CoreAudioKit;
```

```suggestion
  fire = darwin.apple_sdk_11_0.callPackage ../applications/audio/fire {
    inherit (darwin.apple_sdk_11_0.frameworks) Cocoa WebKit CoreServices DiscRecording CoreAudioKit;
```
maybe that fixes the darwin build failure already

---

Also needed some additional dependencies, fixed now.